packages/ui/components/ui/tx/view/memo-view.tsx

```diff
@@ -1,36 +1,32 @@
 import { MemoView } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/core/transaction/v1/transaction_pb';
 import { AddressViewComponent } from './address-view';
-import { ViewBox, ViewSection } from './viewbox';
+import { ViewBox } from './viewbox';
+import { ActionDetails } from './action-details';
 
 export const MemoViewComponent = ({ memo: { memoView } }: { memo: MemoView }) => {
   switch (memoView.case) {
     case 'visible':
       return (
-        <ViewSection heading='Memo'>
-          <ViewBox
-            label='Memo Text'
-            visibleContent={
-              <div>
-                {/* eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing */}
-                {memoView.value.plaintext?.text || (
-                  <span className='italic text-gray-600'>== none ==</span>
-                )}
-              </div>
-            }
-          />
-          <ViewBox
-            label='Sender Return Address'
-            visibleContent={<AddressViewComponent view={memoView.value.plaintext!.returnAddress} />}
-          />
-        </ViewSection>
+        <ViewBox
+          label='Memo'
+          visibleContent={
+            <div className='flex flex-col gap-4'>
+              <ActionDetails>
+                <ActionDetails.Row label='Sender Return Address'>
+                  <AddressViewComponent view={memoView.value.plaintext!.returnAddress} />
+                </ActionDetails.Row>
+                <ActionDetails.Row label='Memo Text'>
+                  <span className='italic' style={{ wordBreak: 'normal' }}>
```

Comment:
Could we get rid of the italic?

Reply:
I did the italics on purpose.  I felt that it was a reasonably unobtrusive way to signal that the text was different from the other text on the page — all of the other words and labels are generated by the frontend, but the memo text is supplied by the counterparty (who is maybe adversarial, eg not escaping it would be bad since they could then inject scripts into the user's page, but even if we're not XSSable it seems good to signal that the text comes from the transaction, not from the UI).

Reply:
Makes sense!